src/FlaUILibrary/flaui/module/element.py

```diff
@@ -456,3 +458,19 @@ def _set_timeout(self, timeout: int):
             timeout (Number): Timeout value in seconds
         """
         self._timeout = timeout
+
+    def _focus_element(self, element, xpath):
+        """Triggers focus action in given element.
+
+        Args:
+            element (Element): The element to be focused
+            xpath (String): XPath of element to be focused
+        
+        Raises:
+            FlaUiError: If the given element is not focusable.
+        """
+        try:
+            element.Focus()
+        except InvalidOperationException:
+            raise FlaUiError(FlaUiError.ElementNotFocusable.format(xpath))

```

Comment:
raise FlaUiError(FlaUiError.ElementNotFocusable.format(xpath)) from None

Why? --> https://realpython.com/python-raise-exception/

You want to hide the original InvalidOperationException so None must be used